lib/routes/seagullApi.js

```diff
@@ -36,9 +36,10 @@ module.exports = function (crudHandler, userApiClient, gatekeeperClient, metrics
   function createDocAndCallback(userId, res, next, cb) {
     crudHandler.createDoc(userId, {}, function (err, result) {
       if (err) {
-        log.error(err, 'Error creating metadata doc');
+        log.error(err, 'createDocAndCallback: Error creating metadata doc');
         if (err.statusCode == 400) {
-          res.send(422);
+	  /* return a 500 code to indicate a temporary server error.  Expect a retry. */
```

Comment:
What will the gateway return on 3 failed retries? Is it worth commenting on that here?

Reply:
The gateway will return exactly what it receives, i.e. a 500.  So I don't think we need to document that.